conf.json

```diff
@@ -1,9 +1,9 @@
-{ settings: 
+{ "settings": 
 	[
 		{
-		  "src_table": "info",
-		  "dest_table": "info",
-		  "column_mapping": {"ID_": "id", "DURATION_": "start_time"},
+		  "src_table": "Camunda_dump_example_csv",
```

Comment:
Here it should be possible to specify the name of the table. Am I correct? If yes, just use an example table name without dump or csv in the name.